src/App.js

```diff
@@ -4,6 +4,8 @@ import Header from "./components/Header";
 import InputPosition from "./components/InputPosition";
 import BtnSubmitPosition from "./components/BtnSubmitPosition";
 import Results from "./components/Results";
+import WeatherList from "./components/WeatherDataList";
+import refreshWeather from "./components/WeatherDataList";
```

Comment:
This will not work. refreshWeather is still the component.
Thus, you can not call the function on button click.